code/modules/mob/say.dm

```diff
@@ -145,6 +145,10 @@
   */
 /mob/proc/get_message_mods(message, list/mods)
 	for(var/I in 1 to MESSAGE_MODS_LENGTH)
+		// Prevents "...text" from being read as a radio message
+		if (length(message) > 1 && message[2] == message[1])
```

Comment:
ok so
string are weird in dm. Message[2] might actually point to first character because utf-8 characters take up to 4 characters (weird). Are you sure it works in this case? You can test it by saying copied utf emote.

Reply:
I'm not able to test this thoroughly at the moment but from what I've tried it should work fine

Once the proc finds a character that isn't a message modifier it returns anyways so I'm not sure if this will be an issue

Reply:
Yeah I fiddled around with it for a bit I'm pretty sure this is fine